### DISCUSSION TITLE
光谷社区第三方 Android 客户端 v2.0 发布 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#214] [Announcements]
https://mazhuang.org/2018/04/30/guanggoo-android-app-new-version/ 

 我最完整的业余作品 光谷社区第三方 Android 客户端 （虽然也还不是很完善）今天发布了重大更新，版本号由 v1.3 更新到 v2.0，欢迎所有身在武汉和心系武汉的朋友们试用体验，也欢迎各路开发者大神们交流指点。

**Comment by IceFlameWorm** [comment]
你好

**Comment by ghost** [comment]
你好，这个页面不支持 {{ }}这种格式？

**Comment by mzlogin** [comment]
@92du 你是指评论还是正文里？ `{{ }}` 会命中 Liquid 标记语法吧，你是想要用这种格式实现什么效果呢？

**Comment by ghost** [comment]
大神，麻烦帮我看看，



1071102039@qq.com
 
发件人： Zhuang Ma
发送时间： 2018-06-15 12:31
收件人： mzlogin/blog-comments
抄送： 92du; Mention
主题： Re: [mzlogin/blog-comments] 光谷社区第三方 Android 客户端 v2.0 发布 — 码志 (#134)
@92du 你是指评论还是正文里？ {{ }} 会命中 Liquid 标记语法吧，你是想要用这种格式实现什么效果呢？

**Comment by mzlogin** [comment]
@92du 

具体点说下你的问题吧~

**Comment by ghost** [comment]
是这样的我希望在正文中显示 {{ NAME }} 这样的源字符，，但是这个在经过渲染后就是空白的了，我应该怎么写，才能在正文中出现这样的 {{ NAME }}效果？


1071102039@qq.com
 
发件人： Zhuang Ma
发送时间： 2018-06-15 12:40
收件人： mzlogin/blog-comments
抄送： 92du; Mention
主题： Re: [mzlogin/blog-comments] 光谷社区第三方 Android 客户端 v2.0 发布 — 码志 (#134)
@92du
具体点说下你的问题吧~

**Comment by mzlogin** [comment]
@92du 

试试这样写：

```
{% raw %}
{{ NAME }}
{% endraw %}
```

**Comment by Xiaocai9527** [comment]
谷歌play竟然给我推荐了这个应用，嘿嘿，我也考虑后期fork参与一下。

**Comment by mzlogin** [comment]
> @xiaokun19931126
> 谷歌play竟然给我推荐了这个应用，嘿嘿，我也考虑后期fork参与一下。

好啊~欢迎欢迎，少年你也是在武汉么？加个微信保持交流吧，我 ID 即微信号。

**Comment by yizibi** [comment]
光谷ios社区咋没人维护了，UI也不精致？

**Comment by mzlogin** [comment]
> @yizibi
> 光谷ios社区咋没人维护了，UI也不精致？

iOS 版是另外一个朋友维护的，估计最近忙吧 :cry:
